src/Domain/Creators/ShapeMapOptions.hpp

```diff
@@ -53,6 +56,61 @@ struct KerrSchildFromBoyerLindquist {
 /// Label for shape map options
 struct Spherical {};
 
+struct YlmsFromFile {
+  struct H5Filename {
+    using type = std::string;
+    static constexpr Options::String help =
+        "Path to the data file containing the ylm coefficients and their "
+        "derivatives.";
+  };
+
+  struct SubfileNames {
+    using type = std::array<std::string, 3>;
```

Comment:
What if I don't have the time derivatives (yet), i.e. just starting from a horizon found in initial data? Should these be optionals?

Reply:
For some reason, option parsing didn't like `std::array<Options::Auto<std::string>, 3>` and I didn't feel like going down the rabbit hole of why options didn't interpret this correctly so I just went with `std::vector<std::string>` instead and using upper/lower bounds on the size. I think it's much less common to need the time deriv of the coefs and not the coefs themselves. If somebody does need that then we can revisit this